portfolio/src/main/webapp/js/comments.js

```diff
@@ -0,0 +1,198 @@
+/**
+ * Retreives comments from server.
+*/
+async function getCommentFromServer(url) {
+  return await fetch(url).then(response => response.json());
+}
+
+/**
+ * Get blobstore upload url from server and set it to form.
+*/
+async function setFormUploadUrl() {
+  const url = await postRequest('comment-upload-url', {redirectUrl: getServletUrl()})
+    .then(response => response.text());
+
+  document.getElementById('comment-form').action = url;
+}
+
+/**
+ * Get get the servlet url of the comment section.
+*/
+function getServletUrl() {
+  return document.getElementById('comment-form').classList[0];
+}
+
+/**
+* Dynamically appends comments from server to DOM.
+*/
+async function printComments() {
+
```

Comment:
Remove the extra empty lines in this method.

Reply:
Fixed it.